src/components/builder.ts

```diff
@@ -306,13 +306,13 @@ export class Builder {
         this.extension.completer.reference.setNumbersFromAuxFile(rootFile)
         this.extension.manager.findAdditionalDependentFilesFromFls(rootFile)
         const configuration = vscode.workspace.getConfiguration('latex-workshop')
-        if (configuration.get('synctex.afterBuild.enabled') as boolean) {
-            this.extension.logger.addLogMessage('SyncTex after build invoked.')
-            this.extension.locator.syncTeX()
-        }
+        // if (configuration.get('synctex.afterBuild.enabled') as boolean) {
+        //     this.extension.logger.addLogMessage('SyncTex after build invoked.')
```

Comment:
Should we copy this line to the corresponding place in `viewer.ts`?

Reply:
Yes but it is not really synctex after build. It gets called every time the viewer is refreshed so even when we first open it. Anyway this might be worth putting a log message.